api/types/client.go

```diff
@@ -269,7 +270,7 @@ type TargetInfo struct {
 }
 
 type VanClientInterface interface {
-	RouterCreate(ctx context.Context, options SiteConfig) error
+	RouterCreate(ctx context.Context, options SiteConfig, loadbalancertimeout int) error
```

Comment:
Maybe you could just set the timeout value at `SiteConfig.Spec.LoadBalancerTimeout`
during skupper init or during site-controller initialization and avoid changing this signature.

Reply:
I did not want to store that information in the skupper internal yaml, because I thought that was not relevant in that file. But probably is a better outcome than changing the signature.

Reply:
I don't think this makes sense as a setting in the skupper-site configmap. Logically, it is really a configuration option for the CLI, not the site. The ideal solution for declarative use would be something more adaptive, or at least asynchronous. So to me, the signature change feels 'cleanest' for a simple improvement on current behaviour.

Reply:
Considering that all fields in the SiteConfig spec are persisted, it makes sense to remove it and simply use as a flag.

Reply:
@nluaces actually I think that is a great idea. Setting a deadline on the context seems a very good approach to allowing a timeout to be specified when using the cli. That same approach could eventually be applied more uniformly.

Reply:
:+1: that is something that we should explore more for request scoped contexts for sure.